modules/mod_ginger_banner/mod_ginger_banner.erl

```diff
@@ -0,0 +1,8 @@
+-module(mod_ginger_banner).
+-author("Driebit <info@driebit.nl>").
+
+-mod_title("Ginger banner module").
+-mod_description("Shows a banner on top of the site, if this is added in the config of the site or trough the rsc banner.").
+-mod_prio(500).
```

Comment:
Should be a *lower* number than mod_ginger_base (which is 250): see also the [docs](http://docs.zotonic.com/en/latest/developer-guide/modules.html#priority).